Exabyte_TeamRoom3.md

```diff
@@ -14,8 +14,16 @@
   - [ ] Other
         - if other specify: ...
 * Experimental Data: 
-  * Describe the classes and types of materials covered. 
-    *  ....
+  * Calculate the properties of the material (DFT)
+    *  Calculation of electronic structure of materials (such as band gap, band structure, Fermi surface)
+    *  Calculation of vibrational properties of materials (such as zero energy)
+    *  Calculation of thermodynamic properties of materials (such as surface energy)
+    *  ......
+  * Machine learning of material data 
+    *  Classify and cluster the experimental data
+    *  Predict material properties
+    *  Create a new training model to train data
+    *  ......
```

Comment:
that is a good point, we can add something like it. I would propose to create two issues, one for adding more details to the DFT as proposed above (user DFT_) and the other issue is about adding new metadata about machine learning specific data and methods.